src/components/CreateArea.jsx

```diff
@@ -1,25 +1,62 @@
-import React from "react";
+import React, { useState } from "react";
+import Note from "./Note";
 
-const CreateArea = (props) => {
+const CreateArea = ({ onAdd }) => {
+  const [note, setNote] = useState({
+    title: "",
+    content: "",
+  });
+
+  function handleChange(e) {
+    const { name, value } = e.target;
+    setNote((preValue) => {
+      return {
+        ...preValue,
+        [name]: value,
+      };
+    });
+  }
+
+  const submitButton = (event) => {
+    onAdd(note);
+    setNote({
+      title: "",
+      content: "",
+    });
+    event.preventDefault();
+  };

```

Comment:
should check if note is empty. Should only allow to add note when title and content are not empty